### PR TITLE
Fix release

### PR DESCRIPTION
**Pull request by ThetaSinner**
### Summary

Includes  #3182

### TODO:
- [ ] CHANGELOG(s) updated with appropriate info
- [ ] Just before pressing the merge button, ensure new entries to CHANGELOG(s) are still under the _UNRELEASED_ heading

### CODE REVIEW

File: crates/aitia/Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0"
 derive_more = { workspace = true }
-holochain_trace = "0.3.0-beta-dev.1"
```
Comment (steveej) [inline crates/aitia/Cargo.toml:15]: i think this become a workspace dependency

**Review by steveej — APPROVED** [review]
:raised_hands:

File: crates/aitia/Cargo.toml (hunk 1 above)
Comment (ThetaSinner) [reply crates/aitia/Cargo.toml:15]: Doesn't look like it, keystore and types moved but not trace

Comment (steveej) [reply crates/aitia/Cargo.toml:15]: oh, i forgot "could". because it's external to the repo

**Pull request merged**